docs/maps/maps-aggregations.asciidoc

```diff
@@ -48,13 +48,13 @@ The cluster location is the weighted centroid for all geo-points in the gridded
 
 *Heat map*:: Creates a <<heatmap-layer, heat map layer>> that clusters the weighted centroids for each gridded cell.
 
-To enable grid aggregation:
+To enable a grid aggregation layer:
 
-. Click *Add layer*, then select the *Grid aggregation* source.
+. Click *Add layer*, then select the *Clusters and grids* layer card or the *Heat map* layer card.
 
 To enable a blended layer that dynamically shows clusters or documents:
 
-. Click *Add layer*, then select the *Documents* source.
+. Click *Add layer*, then select the *Documents* layer card.
 . Configure *Index pattern* and the *Geospatial field*. To enable clustering, the *Geospatial field* must be set to a field mapped as {ref}/geo-point.html[geo_point].
```

Comment:
Should this be:
. Configure the **Index pattern** and the **Geospatial** fields?